  INFO root [2019-09-21 09:28:05,737] - 
  INFO root [2019-09-21 09:28:05,738] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:28:05,739] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:28:05,739] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:28:05,739] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:28:05,750] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51543 )
  INFO root [2019-09-21 09:28:05,750] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51543, SOMAXCONN=5.
  INFO root [2019-09-21 09:28:05,750] - server-configs:
  INFO root [2019-09-21 09:28:05,750] - 	internalTcpAddr : 10.15.26.195:51543
  INFO root [2019-09-21 09:28:05,751] - 	componentID : 5000
  WARN root [2019-09-21 09:28:05,751] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:28:07,255] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 2812 ----
  INFO root [2019-09-21 09:28:07,256] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:28:07,256] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:28:07,256] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:28:07,689] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 09:28:07,689] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:28:14,902] - Components::findComponents: find logger(1)...
  WARN root [2019-09-21 09:28:15,237] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20934, errorno=10048
  INFO root [2019-09-21 09:28:16,240] - Components::findComponents: find logger(2)...
 ERROR root [2019-09-21 09:28:17,741] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:17,743] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:28:17,743] - Components::findComponents: not found logger!
  INFO root [2019-09-21 09:28:18,762] - Components::findComponents: find dbmgr(1)...
 ERROR root [2019-09-21 09:28:20,264] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:20,265] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:21,283] - Components::findComponents: find dbmgr(2)...
 ERROR root [2019-09-21 09:28:22,784] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:22,785] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:23,802] - Components::findComponents: find dbmgr(3)...
 ERROR root [2019-09-21 09:28:25,304] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:25,305] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:26,322] - Components::findComponents: find dbmgr(4)...
 ERROR root [2019-09-21 09:28:27,824] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:27,826] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:28,840] - Components::findComponents: find dbmgr(5)...
 ERROR root [2019-09-21 09:28:30,341] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:30,343] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:31,363] - Components::findComponents: find dbmgr(6)...
 ERROR root [2019-09-21 09:28:32,864] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:32,866] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:33,882] - Components::findComponents: find dbmgr(7)...
 ERROR root [2019-09-21 09:28:35,383] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:35,385] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:36,403] - Components::findComponents: find dbmgr(8)...
 ERROR root [2019-09-21 09:28:37,904] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:37,906] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:38,922] - Components::findComponents: find dbmgr(9)...
 ERROR root [2019-09-21 09:28:40,423] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:40,425] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:41,440] - Components::findComponents: find dbmgr(10)...
 ERROR root [2019-09-21 09:28:42,942] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:42,944] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:43,962] - Components::findComponents: find dbmgr(11)...
 ERROR root [2019-09-21 09:28:45,464] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:45,466] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:46,482] - Components::findComponents: find dbmgr(12)...
 ERROR root [2019-09-21 09:28:47,983] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:47,984] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:49,002] - Components::findComponents: find dbmgr(13)...
 ERROR root [2019-09-21 09:28:50,503] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:50,505] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:51,522] - Components::findComponents: find dbmgr(14)...
 ERROR root [2019-09-21 09:28:53,024] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:53,026] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:54,041] - Components::findComponents: find dbmgr(15)...
 ERROR root [2019-09-21 09:28:55,541] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:55,543] - Components::findComponents: receive error!
  INFO root [2019-09-21 09:28:56,562] - Components::findComponents: find dbmgr(16)...
 ERROR root [2019-09-21 09:28:58,064] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:58,066] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:28:59,083] - Components::findComponents: find dbmgr(17)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:00,586] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:00,588] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:01,602] - Components::findComponents: find dbmgr(18)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:03,105] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:03,107] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:04,122] - Components::findComponents: find dbmgr(19)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:05,625] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:05,628] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:06,641] - Components::findComponents: find dbmgr(20)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:08,146] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:08,148] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:09,162] - Components::findComponents: find dbmgr(21)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:10,665] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:10,667] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:11,682] - Components::findComponents: find dbmgr(22)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:13,185] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:13,187] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:14,202] - Components::findComponents: find dbmgr(23)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:15,710] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:15,712] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:16,722] - Components::findComponents: find dbmgr(24)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:18,225] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:18,227] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:19,240] - Components::findComponents: find dbmgr(25)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:20,744] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:20,746] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:21,762] - Components::findComponents: find dbmgr(26)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:23,265] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:23,266] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:24,282] - Components::findComponents: find dbmgr(27)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:25,786] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:25,788] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:26,802] - Components::findComponents: find dbmgr(28)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:28,304] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:28,307] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:29,322] - Components::findComponents: find dbmgr(29)...
delay time is too long, please check the dbmgr logs!
 ERROR root [2019-09-21 09:29:30,826] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:29:30,827] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:29:31,840] - Components::findComponents: find dbmgr(30)...
delay time is too long, please check the dbmgr logs!
  INFO root [2019-09-21 09:29:34,235] - 
  INFO root [2019-09-21 09:29:34,235] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:29:34,238] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:29:34,238] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:29:34,238] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:29:34,247] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51563 )
  INFO root [2019-09-21 09:29:34,247] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51563, SOMAXCONN=5.
  INFO root [2019-09-21 09:29:34,248] - server-configs:
  INFO root [2019-09-21 09:29:34,248] - 	internalTcpAddr : 10.15.26.195:51563
  INFO root [2019-09-21 09:29:34,248] - 	componentID : 5000
  WARN root [2019-09-21 09:29:34,248] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-21 09:29:34,589] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20891, errorno=10048
  INFO root [2019-09-21 09:29:34,592] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 5028 ----
  INFO root [2019-09-21 09:29:34,594] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:29:34,594] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:29:34,595] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:29:35,015] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 09:29:35,016] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:29:42,157] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:29:42,245] - Components::findComponents: found logger, addr:10.15.26.195:51565
  INFO root [2019-09-21 09:29:42,246] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:29:42,246] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:29:42,247] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 09:29:42,349] - Components::findComponents: found dbmgr, addr:10.15.26.195:51562
  INFO root [2019-09-21 09:29:42,350] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:29:42,350] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:29:42,350] - Components::findComponents: found dbmgr, addr:10.15.26.195:51562
  WARN root [2019-09-21 09:29:43,351] - Forwarding logs to logger[10.15.26.195:51565/0/0/0]...
  INFO root [2019-09-21 09:38:33,461] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [2019-09-21 09:40:36,904] - 
  INFO root [2019-09-21 09:40:36,905] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:40:36,906] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:40:36,906] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:40:36,906] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:40:36,918] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51756 )
  INFO root [2019-09-21 09:40:36,918] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51756, SOMAXCONN=5.
  INFO root [2019-09-21 09:40:36,919] - server-configs:
  INFO root [2019-09-21 09:40:36,919] - 	internalTcpAddr : 10.15.26.195:51756
  INFO root [2019-09-21 09:40:36,919] - 	componentID : 5000
  WARN root [2019-09-21 09:40:36,919] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:40:38,423] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 14140 ----
  INFO root [2019-09-21 09:40:38,424] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:40:38,424] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:40:38,425] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:40:38,881] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 09:40:38,883] - Components::process(): Request for the process of identity...
  WARN root [2019-09-21 09:40:42,335] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20794, errorno=10048
  INFO root [2019-09-21 09:40:46,387] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:40:46,404] - Components::findComponents: found logger, addr:10.15.26.195:51754
  INFO root [2019-09-21 09:40:46,405] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:40:46,405] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:40:46,406] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 09:40:46,411] - Components::findComponents: found dbmgr, addr:10.15.26.195:51759
  INFO root [2019-09-21 09:40:46,411] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:40:46,411] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:40:46,512] - Components::findComponents: found dbmgr, addr:10.15.26.195:51759
  WARN root [2019-09-21 09:40:47,512] - Forwarding logs to logger[10.15.26.195:51754/0/0/0]...
 ERROR root [2019-09-21 09:45:33,914] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=467633612106, lastReceivedTime=467362708914, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-21 09:45:33,918] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:45:33,918] - ==> [2019-09-21 09:45:33 913] TCPPacketReceiver::processPendingEvents(10.15.26.195:51754): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:45:33,918] - ==> [2019-09-21 09:45:33 915] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=467633612106, lastReceivedTime=467362708914, inactivityExceptionPeriod=599500320)
  WARN root [2019-09-21 09:45:34,014] - TCPPacketReceiver::processPendingEvents(10.15.26.195:51759): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:45:34,015] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=467634615451, lastReceivedTime=467364184930, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-21 09:45:34,025] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 09:45:34,028] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 09:45:34,029] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 09:45:34,030] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 09:48:09,552] - 
  INFO root [2019-09-21 09:48:09,553] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:48:09,554] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:48:09,554] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:48:09,554] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:48:09,564] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51842 )
  INFO root [2019-09-21 09:48:09,565] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51842, SOMAXCONN=5.
  INFO root [2019-09-21 09:48:09,565] - server-configs:
  INFO root [2019-09-21 09:48:09,565] - 	internalTcpAddr : 10.15.26.195:51842
  INFO root [2019-09-21 09:48:09,565] - 	componentID : 5000
  WARN root [2019-09-21 09:48:09,567] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:48:11,072] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 10096 ----
  INFO root [2019-09-21 09:48:11,072] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:48:11,073] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:48:11,073] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:48:11,502] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 09:48:11,503] - Components::process(): Request for the process of identity...
  WARN root [2019-09-21 09:48:17,653] - TCPPacketReceiver::processPendingEvents(10.15.26.195:51863): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-21 09:48:18,655] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:48:18,740] - Components::findComponents: found logger, addr:10.15.26.195:51845
  INFO root [2019-09-21 09:48:18,740] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:48:18,740] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:48:18,741] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 09:48:18,745] - Components::findComponents: found dbmgr, addr:10.15.26.195:51846
  INFO root [2019-09-21 09:48:18,746] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:48:18,746] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:48:18,847] - Components::findComponents: found dbmgr, addr:10.15.26.195:51846
  WARN root [2019-09-21 09:48:19,848] - Forwarding logs to logger[10.15.26.195:51845/0/0/0]...
 ERROR root [2019-09-21 09:52:20,732] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=471701786400, lastReceivedTime=471590627073, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-21 09:52:20,740] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:52:20,740] - ==> [2019-09-21 09:52:20 731] TCPPacketReceiver::processPendingEvents(10.15.26.195:51845): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:52:20,741] - ==> [2019-09-21 09:52:20 733] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=471701786400, lastReceivedTime=471590627073, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-21 09:52:20,832] - TCPPacketReceiver::processPendingEvents(10.15.26.195:51846): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:52:20,833] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=471702792465, lastReceivedTime=471590608936, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-21 09:52:20,844] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 09:52:20,855] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 09:52:20,855] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 09:52:20,856] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 09:54:15,544] - 
  INFO root [2019-09-21 09:54:15,551] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:54:15,552] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:54:15,552] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:54:15,553] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:54:15,577] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51948 )
  INFO root [2019-09-21 09:54:15,577] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51948, SOMAXCONN=5.
  INFO root [2019-09-21 09:54:15,578] - server-configs:
  INFO root [2019-09-21 09:54:15,578] - 	internalTcpAddr : 10.15.26.195:51948
  INFO root [2019-09-21 09:54:15,578] - 	componentID : 5000
  WARN root [2019-09-21 09:54:15,583] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-21 09:54:15,981] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20493, errorno=10048
  INFO root [2019-09-21 09:54:15,985] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 13348 ----
  INFO root [2019-09-21 09:54:15,985] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:54:15,985] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:54:15,986] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:54:16,439] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 09:54:16,440] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:54:23,567] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:54:23,667] - Components::findComponents: found logger, addr:10.15.26.195:51946
  INFO root [2019-09-21 09:54:23,667] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:54:23,667] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:54:23,669] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 09:54:23,674] - Components::findComponents: found logger, addr:10.15.26.195:51946
  INFO root [2019-09-21 09:54:23,678] - Components::findComponents: found dbmgr, addr:10.15.26.195:51947
  INFO root [2019-09-21 09:54:23,678] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:54:23,679] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:54:23,782] - Components::findComponents: found dbmgr, addr:10.15.26.195:51947
  WARN root [2019-09-21 09:54:24,782] - Forwarding logs to logger[10.15.26.195:51946/0/0/0]...
 ERROR root [2019-09-21 09:56:00,285] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=473897324212, lastReceivedTime=473749877367, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-21 09:56:00,287] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:56:00,288] - ==> [2019-09-21 09:56:00 285] TCPPacketReceiver::processPendingEvents(10.15.26.195:51946): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:56:00,288] - ==> [2019-09-21 09:56:00 286] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=473897324212, lastReceivedTime=473749877367, inactivityExceptionPeriod=599499745)
  WARN root [2019-09-21 09:56:00,377] - TCPPacketReceiver::processPendingEvents(10.15.26.195:51947): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:56:00,378] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=473898244540, lastReceivedTime=473750516210, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-21 09:56:00,384] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 09:56:00,390] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 09:56:00,391] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 09:56:00,391] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 09:56:02,111] - 
  INFO root [2019-09-21 09:56:02,118] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:56:02,119] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:56:02,119] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:56:02,119] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:56:02,133] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52018 )
  INFO root [2019-09-21 09:56:02,133] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52018, SOMAXCONN=5.
  INFO root [2019-09-21 09:56:02,133] - server-configs:
  INFO root [2019-09-21 09:56:02,133] - 	internalTcpAddr : 10.15.26.195:52018
  INFO root [2019-09-21 09:56:02,134] - 	componentID : 5000
  WARN root [2019-09-21 09:56:02,137] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:56:02,602] - Components::findLogger: found logger, addr:10.15.26.195:52015
  INFO root [2019-09-21 09:56:02,602] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:56:02,602] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:56:02,607] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 13672 ----
  INFO root [2019-09-21 09:56:02,613] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:56:02,614] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:56:02,614] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:56:03,112] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 09:56:03,113] - Components::process(): Request for the process of identity...
  WARN root [2019-09-21 09:56:09,234] - Forwarding logs to logger[10.15.26.195:52015/0/0/0]...
 ERROR root [2019-09-21 09:57:59,386] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=475088330496, lastReceivedTime=474815473564, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-21 09:57:59,388] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:57:59,388] - ==> [2019-09-21 09:57:59 386] TCPPacketReceiver::processPendingEvents(10.15.26.195:52015): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:57:59,388] - ==> [2019-09-21 09:57:59 387] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=475088330496, lastReceivedTime=474815473564, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-21 09:57:59,480] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52013): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:57:59,481] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=475089277239, lastReceivedTime=474814634293, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-21 09:57:59,498] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 09:57:59,500] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 09:57:59,501] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 09:57:59,501] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 09:58:01,165] - 
  INFO root [2019-09-21 09:58:01,170] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:58:01,171] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:58:01,171] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:58:01,171] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:58:01,221] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52075 )
  INFO root [2019-09-21 09:58:01,221] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52075, SOMAXCONN=5.
  INFO root [2019-09-21 09:58:01,222] - server-configs:
  INFO root [2019-09-21 09:58:01,222] - 	internalTcpAddr : 10.15.26.195:52075
  INFO root [2019-09-21 09:58:01,222] - 	componentID : 5000
  WARN root [2019-09-21 09:58:01,222] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:58:02,729] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 14332 ----
  INFO root [2019-09-21 09:58:02,730] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:58:02,730] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:58:02,731] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:58:03,173] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 09:58:03,174] - Components::process(): Request for the process of identity...
  WARN root [2019-09-21 09:58:09,315] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52089): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-21 09:58:10,332] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:58:10,409] - Components::findComponents: found logger, addr:10.15.26.195:52074
  INFO root [2019-09-21 09:58:10,410] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:58:10,410] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:58:10,411] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 09:58:10,515] - Components::findComponents: found dbmgr, addr:10.15.26.195:52070
  INFO root [2019-09-21 09:58:10,515] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:58:10,515] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:58:10,515] - Components::findComponents: found dbmgr, addr:10.15.26.195:52070
  WARN root [2019-09-21 09:58:11,515] - Forwarding logs to logger[10.15.26.195:52074/0/0/0]...
 ERROR root [2019-09-21 09:59:51,502] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476209484916, lastReceivedTime=476006453206, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-21 09:59:51,504] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:59:51,504] - ==> [2019-09-21 09:59:51 500] TCPPacketReceiver::processPendingEvents(10.15.26.195:52074): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:59:51,504] - ==> [2019-09-21 09:59:51 503] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476209484916, lastReceivedTime=476006453206, inactivityExceptionPeriod=599499745)
  WARN root [2019-09-21 09:59:51,601] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52070): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:59:51,603] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476210499230, lastReceivedTime=476006153512, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-21 09:59:51,634] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 09:59:51,641] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 09:59:51,643] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 09:59:51,644] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 10:01:45,633] - 
  INFO root [2019-09-21 10:01:45,634] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:01:45,635] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:01:45,635] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:01:45,635] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:01:45,644] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52162 )
  INFO root [2019-09-21 10:01:45,644] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52162, SOMAXCONN=5.
  INFO root [2019-09-21 10:01:45,645] - server-configs:
  INFO root [2019-09-21 10:01:45,645] - 	internalTcpAddr : 10.15.26.195:52162
  INFO root [2019-09-21 10:01:45,645] - 	componentID : 5000
  WARN root [2019-09-21 10:01:45,645] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 10:01:47,149] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 8092 ----
  INFO root [2019-09-21 10:01:47,150] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:01:47,150] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:01:47,151] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:01:47,570] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 10:01:47,570] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 10:01:54,713] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 10:01:54,745] - Components::findComponents: found logger, addr:10.15.26.195:52158
  INFO root [2019-09-21 10:01:54,746] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:01:54,746] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:01:54,747] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 10:01:54,756] - Components::findComponents: found logger, addr:10.15.26.195:52158
  INFO root [2019-09-21 10:01:54,759] - Components::findComponents: found dbmgr, addr:10.15.26.195:52159
  INFO root [2019-09-21 10:01:54,760] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:01:54,760] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:01:54,860] - Components::findComponents: found dbmgr, addr:10.15.26.195:52159
  WARN root [2019-09-21 10:01:55,862] - Forwarding logs to logger[10.15.26.195:52158/0/0/0]...
 ERROR root [2019-09-21 10:05:33,656] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=479631028078, lastReceivedTime=479450318916, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-21 10:05:33,661] - The following logs sent to logger failed:
  WARN root [2019-09-21 10:05:33,662] - ==> [2019-09-21 10:05:33 650] TCPPacketReceiver::processPendingEvents(10.15.26.195:52158): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:05:33,662] - ==> [2019-09-21 10:05:33 657] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=479631028078, lastReceivedTime=479450318916, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-21 10:05:33,796] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52159): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:05:33,806] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=479632522555, lastReceivedTime=479450396309, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-21 10:05:33,840] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 10:05:33,840] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 10:05:33,840] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 10:05:33,841] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 10:06:38,550] - 
  INFO root [2019-09-21 10:06:38,555] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:06:38,556] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:06:38,556] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:06:38,557] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:06:38,579] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52224 )
  INFO root [2019-09-21 10:06:38,581] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52224, SOMAXCONN=5.
  INFO root [2019-09-21 10:06:38,581] - server-configs:
  INFO root [2019-09-21 10:06:38,581] - 	internalTcpAddr : 10.15.26.195:52224
  INFO root [2019-09-21 10:06:38,581] - 	componentID : 5000
  WARN root [2019-09-21 10:06:38,584] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 10:06:40,089] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 6024 ----
  INFO root [2019-09-21 10:06:40,090] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:06:40,090] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:06:40,090] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:06:40,533] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 10:06:40,533] - Components::process(): Request for the process of identity...
  WARN root [2019-09-21 10:06:43,994] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20700, errorno=10048
  INFO root [2019-09-21 10:06:48,065] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 10:06:48,105] - Components::findComponents: found logger, addr:10.15.26.195:52222
  INFO root [2019-09-21 10:06:48,105] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:06:48,105] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:06:48,106] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 10:06:48,210] - Components::findComponents: found dbmgr, addr:10.15.26.195:52223
  INFO root [2019-09-21 10:06:48,210] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:06:48,210] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:06:48,210] - Components::findComponents: found dbmgr, addr:10.15.26.195:52223
  WARN root [2019-09-21 10:06:49,210] - Forwarding logs to logger[10.15.26.195:52222/0/0/0]...
 ERROR root [2019-09-21 10:27:24,722] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=492741686748, lastReceivedTime=492579514817, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-21 10:27:24,723] - The following logs sent to logger failed:
  WARN root [2019-09-21 10:27:24,724] - ==> [2019-09-21 10:27:24 719] TCPPacketReceiver::processPendingEvents(10.15.26.195:52222): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:27:24,724] - ==> [2019-09-21 10:27:24 723] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=492741686748, lastReceivedTime=492579514817, inactivityExceptionPeriod=599498785)
  WARN root [2019-09-21 10:27:24,806] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52223): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:27:24,809] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=492742549544, lastReceivedTime=492579857732, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-21 10:27:24,814] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 10:27:24,815] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 10:27:24,815] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 10:27:24,816] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 10:52:47,037] - 
  INFO root [2019-09-21 10:52:47,038] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:52:47,039] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:52:47,039] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:52:47,040] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:52:47,078] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52805 )
  INFO root [2019-09-21 10:52:47,078] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52805, SOMAXCONN=5.
  INFO root [2019-09-21 10:52:47,078] - server-configs:
  INFO root [2019-09-21 10:52:47,078] - 	internalTcpAddr : 10.15.26.195:52805
  INFO root [2019-09-21 10:52:47,078] - 	componentID : 5000
  WARN root [2019-09-21 10:52:47,083] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 10:52:47,517] - Components::findLogger: found logger, addr:10.15.26.195:52800
  INFO root [2019-09-21 10:52:47,518] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:52:47,518] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:52:47,523] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 8600 ----
  INFO root [2019-09-21 10:52:47,523] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:52:47,523] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:52:47,524] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:52:48,037] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 10:52:48,046] - Components::process(): Request for the process of identity...
  WARN root [2019-09-21 10:52:54,213] - Forwarding logs to logger[10.15.26.195:52800/0/0/0]...
 ERROR root [2019-09-21 10:54:19,519] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=508889662709, lastReceivedTime=508864546720, inactivityExceptionPeriod=599499041)
  INFO root [2019-09-21 10:54:19,523] - The following logs sent to logger failed:
  WARN root [2019-09-21 10:54:19,524] - ==> [2019-09-21 10:54:19 515] TCPPacketReceiver::processPendingEvents(10.15.26.195:52800): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:54:19,524] - ==> [2019-09-21 10:54:19 521] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=508889662709, lastReceivedTime=508864546720, inactivityExceptionPeriod=599499041)
  WARN root [2019-09-21 10:54:19,621] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52804): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:54:19,623] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=508890697932, lastReceivedTime=508865138396, inactivityExceptionPeriod=599499041)
  INFO root [2019-09-21 10:54:19,641] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 10:54:19,652] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 10:54:19,652] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 10:54:19,652] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 10:54:21,540] - 
  INFO root [2019-09-21 10:54:21,555] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:54:21,556] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:54:21,556] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:54:21,556] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:54:21,573] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52884 )
  INFO root [2019-09-21 10:54:21,573] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52884, SOMAXCONN=5.
  INFO root [2019-09-21 10:54:21,573] - server-configs:
  INFO root [2019-09-21 10:54:21,574] - 	internalTcpAddr : 10.15.26.195:52884
  INFO root [2019-09-21 10:54:21,574] - 	componentID : 5000
  WARN root [2019-09-21 10:54:21,575] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 10:54:23,083] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 15164 ----
  INFO root [2019-09-21 10:54:23,083] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:54:23,084] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:54:23,084] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:54:23,569] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 10:54:23,570] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 10:54:30,725] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 10:54:30,815] - Components::findComponents: found logger, addr:10.15.26.195:52880
  INFO root [2019-09-21 10:54:30,815] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:54:30,816] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:54:30,817] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 10:54:30,921] - Components::findComponents: found dbmgr, addr:10.15.26.195:52882
  INFO root [2019-09-21 10:54:30,921] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:54:30,921] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:54:30,921] - Components::findComponents: found dbmgr, addr:10.15.26.195:52882
  WARN root [2019-09-21 10:54:31,921] - Forwarding logs to logger[10.15.26.195:52880/0/0/0]...
 ERROR root [2019-09-21 11:09:42,239] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=518116851364, lastReceivedTime=517908114329, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-21 11:09:42,245] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:09:42,245] - ==> [2019-09-21 11:09:42 238] TCPPacketReceiver::processPendingEvents(10.15.26.195:52880): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:09:42,245] - ==> [2019-09-21 11:09:42 239] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=518116851364, lastReceivedTime=517908114329, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-21 11:09:42,343] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52882): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:09:42,345] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=518117926277, lastReceivedTime=517909703831, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-21 11:09:42,375] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 11:09:42,378] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 11:09:42,378] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 11:09:42,378] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 11:10:44,805] - 
  INFO root [2019-09-21 11:10:44,806] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:10:44,806] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:10:44,807] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:10:44,807] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:10:44,832] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53005 )
  INFO root [2019-09-21 11:10:44,832] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53005, SOMAXCONN=5.
  INFO root [2019-09-21 11:10:44,833] - server-configs:
  INFO root [2019-09-21 11:10:44,833] - 	internalTcpAddr : 10.15.26.195:53005
  INFO root [2019-09-21 11:10:44,833] - 	componentID : 5000
  WARN root [2019-09-21 11:10:44,834] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 11:10:46,348] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 4160 ----
  INFO root [2019-09-21 11:10:46,348] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:10:46,349] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:10:46,349] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:10:46,768] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 11:10:46,769] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:10:53,910] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:10:53,957] - Components::findComponents: found logger, addr:10.15.26.195:53004
  INFO root [2019-09-21 11:10:53,958] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:10:53,958] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:10:53,959] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 11:10:54,066] - Components::findComponents: found dbmgr, addr:10.15.26.195:53003
  INFO root [2019-09-21 11:10:54,066] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:10:54,066] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:10:54,067] - Components::findComponents: found dbmgr, addr:10.15.26.195:53003
  WARN root [2019-09-21 11:10:55,069] - Forwarding logs to logger[10.15.26.195:53004/0/0/0]...
 ERROR root [2019-09-21 11:13:18,264] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520277106874, lastReceivedTime=520242450792, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-21 11:13:18,271] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:13:18,271] - ==> [2019-09-21 11:13:18 262] TCPPacketReceiver::processPendingEvents(10.15.26.195:53004): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:13:18,271] - ==> [2019-09-21 11:13:18 265] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520277106874, lastReceivedTime=520242450792, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-21 11:13:18,362] - TCPPacketReceiver::processPendingEvents(10.15.26.195:53003): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:13:18,364] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520278111963, lastReceivedTime=520242410434, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-21 11:13:18,378] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 11:13:18,379] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 11:13:18,379] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 11:13:18,380] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 11:13:20,080] - 
  INFO root [2019-09-21 11:13:20,081] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:13:20,082] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:13:20,082] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:13:20,083] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:13:20,110] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53064 )
  INFO root [2019-09-21 11:13:20,110] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53064, SOMAXCONN=5.
  INFO root [2019-09-21 11:13:20,110] - server-configs:
  INFO root [2019-09-21 11:13:20,110] - 	internalTcpAddr : 10.15.26.195:53064
  INFO root [2019-09-21 11:13:20,119] - 	componentID : 5000
  WARN root [2019-09-21 11:13:20,125] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-21 11:13:20,665] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20871, errorno=10048
  INFO root [2019-09-21 11:13:20,669] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 15548 ----
  INFO root [2019-09-21 11:13:20,669] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:13:20,669] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:13:20,670] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:13:21,094] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 11:13:21,095] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:13:28,286] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:13:28,365] - Components::findComponents: found logger, addr:10.15.26.195:53069
  INFO root [2019-09-21 11:13:28,365] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:13:28,365] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:13:28,366] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 11:13:28,469] - Components::findComponents: found dbmgr, addr:10.15.26.195:53063
  INFO root [2019-09-21 11:13:28,469] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:13:28,469] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:13:28,470] - Components::findComponents: found dbmgr, addr:10.15.26.195:53063
  WARN root [2019-09-21 11:13:29,471] - Forwarding logs to logger[10.15.26.195:53069/0/0/0]...
 ERROR root [2019-09-21 11:16:26,657] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522161037782, lastReceivedTime=522096293105, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-21 11:16:26,660] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:16:26,660] - ==> [2019-09-21 11:16:26 648] TCPPacketReceiver::processPendingEvents(10.15.26.195:53069): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:16:26,661] - ==> [2019-09-21 11:16:26 658] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522161037782, lastReceivedTime=522096293105, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-21 11:16:26,749] - TCPPacketReceiver::processPendingEvents(10.15.26.195:53063): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:16:26,755] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522162016998, lastReceivedTime=522094875220, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-21 11:16:26,781] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 11:16:26,794] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 11:16:26,794] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 11:16:26,794] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 11:16:31,286] - 
  INFO root [2019-09-21 11:16:31,296] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:16:31,297] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:16:31,297] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:16:31,297] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:16:31,344] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53123 )
  INFO root [2019-09-21 11:16:31,344] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53123, SOMAXCONN=5.
  INFO root [2019-09-21 11:16:31,344] - server-configs:
  INFO root [2019-09-21 11:16:31,344] - 	internalTcpAddr : 10.15.26.195:53123
  INFO root [2019-09-21 11:16:31,344] - 	componentID : 5000
  WARN root [2019-09-21 11:16:31,346] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 11:16:32,855] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 14016 ----
  INFO root [2019-09-21 11:16:32,865] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:16:32,865] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:16:32,866] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:16:33,357] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 11:16:33,358] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:16:40,508] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:16:40,537] - Components::findComponents: found logger, addr:10.15.26.195:53118
  INFO root [2019-09-21 11:16:40,537] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:16:40,537] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:16:40,538] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 11:16:40,543] - Components::findComponents: found logger, addr:10.15.26.195:53118
  INFO root [2019-09-21 11:16:40,546] - Components::findComponents: found dbmgr, addr:10.15.26.195:53122
  INFO root [2019-09-21 11:16:40,546] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:16:40,546] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:16:40,647] - Components::findComponents: found dbmgr, addr:10.15.26.195:53122
  WARN root [2019-09-21 11:16:41,647] - Forwarding logs to logger[10.15.26.195:53118/0/0/0]...
 ERROR root [2019-09-21 11:17:01,827] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522512730288, lastReceivedTime=522506264877, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-21 11:17:01,828] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:17:01,829] - ==> [2019-09-21 11:17:01 824] TCPPacketReceiver::processPendingEvents(10.15.26.195:53118): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:17:01,829] - ==> [2019-09-21 11:17:01 827] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522512730288, lastReceivedTime=522506264877, inactivityExceptionPeriod=599499489)
  WARN root [2019-09-21 11:17:01,912] - TCPPacketReceiver::processPendingEvents(10.15.26.195:53122): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:17:01,913] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522513604851, lastReceivedTime=522508011666, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-21 11:17:01,919] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 11:17:01,922] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 11:17:01,922] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 11:17:01,923] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 11:17:03,567] - 
  INFO root [2019-09-21 11:17:03,571] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:17:03,572] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:17:03,572] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:17:03,572] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:17:03,598] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53156 )
  INFO root [2019-09-21 11:17:03,598] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53156, SOMAXCONN=5.
  INFO root [2019-09-21 11:17:03,598] - server-configs:
  INFO root [2019-09-21 11:17:03,619] - 	internalTcpAddr : 10.15.26.195:53156
  INFO root [2019-09-21 11:17:03,620] - 	componentID : 5000
  WARN root [2019-09-21 11:17:03,636] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 11:17:05,155] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 13524 ----
  INFO root [2019-09-21 11:17:05,155] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:17:05,156] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:17:05,156] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:17:05,663] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 11:17:05,664] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:17:12,817] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:17:12,915] - Components::findComponents: found logger, addr:10.15.26.195:53153
  INFO root [2019-09-21 11:17:12,915] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:17:12,916] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:17:12,918] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 11:17:13,020] - Components::findComponents: found dbmgr, addr:10.15.26.195:53158
  INFO root [2019-09-21 11:17:13,020] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:17:13,020] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:17:13,021] - Components::findComponents: found dbmgr, addr:10.15.26.195:53158
  WARN root [2019-09-21 11:17:14,022] - Forwarding logs to logger[10.15.26.195:53153/0/0/0]...
 ERROR root [2019-09-21 11:20:54,431] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=524838778216, lastReceivedTime=524629251132, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-21 11:20:54,433] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:20:54,433] - ==> [2019-09-21 11:20:54 428] TCPPacketReceiver::processPendingEvents(10.15.26.195:53153): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:20:54,434] - ==> [2019-09-21 11:20:54 432] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=524838778216, lastReceivedTime=524629251132, inactivityExceptionPeriod=599499617)
  WARN root [2019-09-21 11:20:54,529] - TCPPacketReceiver::processPendingEvents(10.15.26.195:53158): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:20:54,531] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=524839772087, lastReceivedTime=524631083745, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-21 11:20:54,554] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 11:20:54,555] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 11:20:54,555] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 11:20:54,556] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 14:44:23,569] - 
  INFO root [2019-09-21 14:44:23,627] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 14:44:23,628] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 14:44:23,628] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 14:44:23,628] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 14:44:23,642] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:54430 )
  INFO root [2019-09-21 14:44:23,642] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:54430, SOMAXCONN=5.
  INFO root [2019-09-21 14:44:23,642] - server-configs:
  INFO root [2019-09-21 14:44:23,642] - 	internalTcpAddr : 10.15.26.195:54430
  INFO root [2019-09-21 14:44:23,643] - 	componentID : 5000
  WARN root [2019-09-21 14:44:23,643] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 14:44:25,148] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 14120 ----
  INFO root [2019-09-21 14:44:25,151] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 14:44:25,151] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 14:44:25,152] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 14:44:25,676] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 14:44:25,677] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 14:44:32,790] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 14:44:32,840] - Components::findComponents: found logger, addr:10.15.26.195:54427
  INFO root [2019-09-21 14:44:32,840] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 14:44:32,840] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 14:44:32,841] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 14:44:32,945] - Components::findComponents: found dbmgr, addr:10.15.26.195:54424
  INFO root [2019-09-21 14:44:32,945] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 14:44:32,945] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 14:44:32,946] - Components::findComponents: found dbmgr, addr:10.15.26.195:54424
  WARN root [2019-09-21 14:44:33,946] - Forwarding logs to logger[10.15.26.195:54427/0/0/0]...
 ERROR root [2019-09-21 14:54:59,595] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=653385838334, lastReceivedTime=653324851371, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-21 14:54:59,599] - The following logs sent to logger failed:
  WARN root [2019-09-21 14:54:59,600] - ==> [2019-09-21 14:54:59 594] TCPPacketReceiver::processPendingEvents(10.15.26.195:54427): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 14:54:59,600] - ==> [2019-09-21 14:54:59 595] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=653385838334, lastReceivedTime=653324851371, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-21 14:54:59,708] - TCPPacketReceiver::processPendingEvents(10.15.26.195:54424): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 14:54:59,709] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=653386983548, lastReceivedTime=653322492699, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-21 14:54:59,726] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 14:54:59,728] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 14:54:59,729] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 14:54:59,729] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 15:02:26,986] - 
  INFO root [2019-09-21 15:02:26,987] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:02:26,988] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:02:26,988] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:02:26,989] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:02:27,003] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:54609 )
  INFO root [2019-09-21 15:02:27,004] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:54609, SOMAXCONN=5.
  INFO root [2019-09-21 15:02:27,004] - server-configs:
  INFO root [2019-09-21 15:02:27,004] - 	internalTcpAddr : 10.15.26.195:54609
  INFO root [2019-09-21 15:02:27,004] - 	componentID : 5000
  WARN root [2019-09-21 15:02:27,004] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 15:02:28,510] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 15860 ----
  INFO root [2019-09-21 15:02:28,511] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:02:28,511] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:02:28,511] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:02:28,935] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 15:02:28,945] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 15:02:36,153] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:02:36,252] - Components::findComponents: found logger, addr:10.15.26.195:54614
  INFO root [2019-09-21 15:02:36,252] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:02:36,252] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:02:36,254] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 15:02:36,357] - Components::findComponents: found dbmgr, addr:10.15.26.195:54610
  INFO root [2019-09-21 15:02:36,357] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:02:36,357] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:02:36,357] - Components::findComponents: found dbmgr, addr:10.15.26.195:54610
  WARN root [2019-09-21 15:02:37,357] - Forwarding logs to logger[10.15.26.195:54614/0/0/0]...
 ERROR root [2019-09-21 15:11:42,112] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=663411008840, lastReceivedTime=663260761356, inactivityExceptionPeriod=599498210)
  INFO root [2019-09-21 15:11:42,114] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:11:42,114] - ==> [2019-09-21 15:11:42 111] TCPPacketReceiver::processPendingEvents(10.15.26.195:54614): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:11:42,114] - ==> [2019-09-21 15:11:42 112] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=663411008840, lastReceivedTime=663260761356, inactivityExceptionPeriod=599498210)
  WARN root [2019-09-21 15:11:42,202] - TCPPacketReceiver::processPendingEvents(10.15.26.195:54610): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:11:42,204] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=663411941934, lastReceivedTime=663259937031, inactivityExceptionPeriod=599498210)
  INFO root [2019-09-21 15:11:42,209] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 15:11:42,210] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 15:11:42,210] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 15:11:42,210] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 15:11:43,692] - 
  INFO root [2019-09-21 15:11:43,693] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:11:43,694] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:11:43,694] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:11:43,694] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:11:43,706] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:54718 )
  INFO root [2019-09-21 15:11:43,706] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:54718, SOMAXCONN=5.
  INFO root [2019-09-21 15:11:43,706] - server-configs:
  INFO root [2019-09-21 15:11:43,707] - 	internalTcpAddr : 10.15.26.195:54718
  INFO root [2019-09-21 15:11:43,707] - 	componentID : 5000
  WARN root [2019-09-21 15:11:43,708] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-21 15:11:44,158] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20170, errorno=10048
  INFO root [2019-09-21 15:11:44,162] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 396 ----
  INFO root [2019-09-21 15:11:44,162] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:11:44,163] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:11:44,163] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:11:44,584] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 15:11:44,584] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 15:11:51,704] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:11:51,776] - Components::findComponents: found logger, addr:10.15.26.195:54721
  INFO root [2019-09-21 15:11:51,776] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:11:51,776] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:11:51,778] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 15:11:51,881] - Components::findComponents: found dbmgr, addr:10.15.26.195:54719
  INFO root [2019-09-21 15:11:51,881] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:11:51,881] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:11:51,882] - Components::findComponents: found dbmgr, addr:10.15.26.195:54719
  WARN root [2019-09-21 15:11:52,882] - Forwarding logs to logger[10.15.26.195:54721/0/0/0]...
 ERROR root [2019-09-21 15:22:59,160] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=670181491762, lastReceivedTime=670027387865, inactivityExceptionPeriod=599500192)
  INFO root [2019-09-21 15:22:59,163] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:22:59,163] - ==> [2019-09-21 15:22:59 159] TCPPacketReceiver::processPendingEvents(10.15.26.195:54721): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:22:59,163] - ==> [2019-09-21 15:22:59 160] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=670181491762, lastReceivedTime=670027387865, inactivityExceptionPeriod=599500192)
  WARN root [2019-09-21 15:22:59,239] - TCPPacketReceiver::processPendingEvents(10.15.26.195:54719): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:22:59,240] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=670182291797, lastReceivedTime=670026839330, inactivityExceptionPeriod=599500192)
  INFO root [2019-09-21 15:22:59,242] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 15:22:59,244] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 15:22:59,245] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 15:22:59,245] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 15:35:45,356] - 
  INFO root [2019-09-21 15:35:45,357] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:35:45,358] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:35:45,358] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:35:45,358] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:35:45,368] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:55690 )
  INFO root [2019-09-21 15:35:45,368] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:55690, SOMAXCONN=5.
  INFO root [2019-09-21 15:35:45,368] - server-configs:
  INFO root [2019-09-21 15:35:45,372] - 	internalTcpAddr : 10.15.26.195:55690
  INFO root [2019-09-21 15:35:45,372] - 	componentID : 5000
  WARN root [2019-09-21 15:35:45,372] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 15:35:46,878] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 9232 ----
  INFO root [2019-09-21 15:35:46,878] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:35:46,879] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:35:46,879] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:35:47,286] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 15:35:47,286] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 15:35:54,420] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:35:54,432] - Components::findComponents: found logger, addr:10.15.26.195:55689
  INFO root [2019-09-21 15:35:54,433] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:35:54,433] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:35:54,434] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 15:35:54,537] - Components::findComponents: found dbmgr, addr:10.15.26.195:55691
  INFO root [2019-09-21 15:35:54,537] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:35:54,537] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:35:54,543] - Components::findComponents: found dbmgr, addr:10.15.26.195:55691
  WARN root [2019-09-21 15:35:55,544] - Forwarding logs to logger[10.15.26.195:55689/0/0/0]...
 ERROR root [2019-09-21 15:39:58,335] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=680373245712, lastReceivedTime=680243408552, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-21 15:39:58,341] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:39:58,341] - ==> [2019-09-21 15:39:58 334] TCPPacketReceiver::processPendingEvents(10.15.26.195:55689): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:39:58,341] - ==> [2019-09-21 15:39:58 336] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=680373245712, lastReceivedTime=680243408552, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-21 15:39:58,431] - TCPPacketReceiver::processPendingEvents(10.15.26.195:55691): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:39:58,434] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=680374241186, lastReceivedTime=680243920737, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-21 15:39:58,443] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 15:39:58,447] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 15:39:58,448] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 15:39:58,462] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-21 15:40:00,084] - 
  INFO root [2019-09-21 15:40:00,102] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:40:00,112] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:40:00,113] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:40:00,113] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:40:00,297] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:55784 )
  INFO root [2019-09-21 15:40:00,297] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:55784, SOMAXCONN=5.
  INFO root [2019-09-21 15:40:00,297] - server-configs:
  INFO root [2019-09-21 15:40:00,297] - 	internalTcpAddr : 10.15.26.195:55784
  INFO root [2019-09-21 15:40:00,297] - 	componentID : 5000
  WARN root [2019-09-21 15:40:00,318] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 15:40:01,836] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 4868 ----
  INFO root [2019-09-21 15:40:01,855] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:40:01,857] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:40:01,857] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:40:02,304] - ---- baseappmgr is running ----
 DEBUG root [2019-09-21 15:40:02,305] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 15:40:09,479] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:40:09,540] - Components::findComponents: found logger, addr:10.15.26.195:55781
  INFO root [2019-09-21 15:40:09,540] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:40:09,540] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:40:09,542] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-21 15:40:09,549] - Components::findComponents: found dbmgr, addr:10.15.26.195:55780
  INFO root [2019-09-21 15:40:09,549] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:40:09,550] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:40:09,653] - Components::findComponents: found dbmgr, addr:10.15.26.195:55780
  WARN root [2019-09-21 15:40:10,655] - Forwarding logs to logger[10.15.26.195:55781/0/0/0]...
 ERROR root [2019-09-21 15:55:18,420] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=689574101355, lastReceivedTime=689391129261, inactivityExceptionPeriod=599500448)
  INFO root [2019-09-21 15:55:18,422] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:55:18,422] - ==> [2019-09-21 15:55:18 418] TCPPacketReceiver::processPendingEvents(10.15.26.195:55781): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:55:18,422] - ==> [2019-09-21 15:55:18 421] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=689574101355, lastReceivedTime=689391129261, inactivityExceptionPeriod=599500448)
  WARN root [2019-09-21 15:55:18,585] - TCPPacketReceiver::processPendingEvents(10.15.26.195:55780): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:55:18,586] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=689575761767, lastReceivedTime=689391244003, inactivityExceptionPeriod=599500448)
  INFO root [2019-09-21 15:55:18,635] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-21 15:55:18,636] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-21 15:55:18,636] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-21 15:55:18,637] - ThreadPool::destroy(): starting size 2.
